addons/web_editor/static/src/js/backend/fields.js

```diff
@@ -30,7 +30,7 @@ var FieldTextHtmlSimple = basic_fields.DebouncedField.extend(TranslatableFieldMi
      * @override
      */
     start: function () {
-        new SummernoteManager(this);
+        this.SummernoteManager = new SummernoteManager(this);
```

Comment:
Name it `_summernoteManager`, since you instantiate it it should not start with an uppercase letter.